install.sh

```diff
@@ -295,6 +295,8 @@ while true; do
 
 done
 
+export AWS_DEFAULT_REGION;
```

Comment:
why is this required?

Reply:
In few cases if `AWS_DEFAULT_REGION` is not exported by the user. CDK is not populating `CDK_DEFAULT_REGION`. So, just to ignore the case where the user didn't export `AWS_DEFAULT_REGION` I am doing it in the script.